src/NzbDrone.Core/Notifications/Plex/Server/PlexSectionItem.cs

```diff
@@ -6,9 +6,11 @@ namespace NzbDrone.Core.Notifications.Plex.Server
     public class PlexSectionItem
     {
         [JsonProperty("ratingKey")]
-        public int Id { get; set; }
+        public string Id { get; set; }
```

Comment:
`ratingKey` is considered a string in the Plex API. And the values coming back from providers are a uuid rather than an int. So this change is required to make the list import work.